ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumptionSpec.scala

```diff
@@ -97,6 +99,11 @@ class KeyValueConsumptionSpec extends AnyWordSpec with Matchers {
           Assertions(update =>
             inside(update) { case Some(CommandRejected(_, _, FinalReason(status))) =>
               status.code shouldBe Code.ALREADY_EXISTS.value
+              status.details shouldBe Seq(
+                AnyProto.pack[ErrorInfo](
+                  ErrorInfo(metadata = Map(GrpcStatuses.DefiniteAnswerKey -> "false"))
+                )
+              )
               ()
```

Comment:
It would also make sense to add a test case variant that has `transactionRejectionEntry.definiteAnswer = true`.